scipy/conftest.py

```diff
@@ -228,8 +232,19 @@ def xp(request):
     else:
         yield request.param
 
+# OBSOLETE, pending removal
 array_api_compatible = pytest.mark.array_api_compatible
 
+@pytest.fixture
+def skip_xp_backends():
+    pass
+
+@pytest.fixture
+def xfail_xp_backends():
+    pass
+# /OBSOLETE, pending removal
```

Comment:
Have to admit I don't understand what's going on here. Why there's a need to add empty fixtures. Or does their existence somehow interact with what you added inside of the `xp` fixture?

Reply:
There is need for an empty fixture for as long as there are any tests decorated with `@pytest.marks.usefixtures("skip_xp_backends")`.